linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -79,6 +84,13 @@ private[telemeter] object UsageDataTelemeter {
   def mkContainerManager: Option[String] =
     sys.env.get("MESOS_TASK_ID").map(_ => "mesos")
 
+  def mkStartTime: Option[String] = {
+    val tz = TimeZone.getTimeZone("UTC")
+    val df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'")
+    df.setTimeZone(tz)
+    Some(df.format(new Date()))
```

Comment:
This will get the current time when the client is created which isn't exactly the same as the linkerd start time (though maybe it's close enough?).  What about using the jvm uptime metric?  

Also: would we rather send `startTime` or `uptime` in the payload?  I guess one can always derive the other if the message timestamp is known.